clang/include/clang/Basic/MacroBuilder.h

```diff
@@ -26,7 +26,7 @@ class MacroBuilder {
   MacroBuilder(raw_ostream &Output) : Out(Output) {}
```

Comment:
This suggestion fixes warnings I see:

```suggestion
  MacroBuilder(raw_ostream &Output) : Out(Output) {}
  virtual ~MacroBuilder() {}
```

Building with clang-8, I get a screen full of warnings of the form:

```
class MacroBuilder {
      ^
1 warning generated.
[576/625] Building CXX object tools/clang/lib/Basic/CMakeFiles/obj.clangBasic.dir/Targets.cpp.o
In file included from /home/petwal01/classic-flang-llvm-project/clang/lib/Basic/Targets.cpp:14:
In file included from /home/petwal01/classic-flang-llvm-project/clang/lib/Basic/Targets.h:19:
/home/petwal01/classic-flang-llvm-project/clang/include/clang/Basic/MacroBuilder.h:23:7: warning: 'clang::MacroBuilder' has virtual functions but non-virtual destructor [-Wnon-virtual-dtor]
class MacroBuilder {
      ^
1 warning generated.
```

My cmake line looks like so:
```
CC=clang-8 CXX=clang++-8 cmake ../llvm -GNinja -DLLVM_USE_LINKER=lld -DLLVM_ENABLE_PROJECTS='clang' -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=On -DLLVM_ENABLE_CLASSIC_FLANG=On
```

Reply:
Applied your fix.